app/views/early_bird_dashboard/index.html.slim

```diff
@@ -63,4 +64,13 @@
               - else
                 span.constituency #{question.member_constituency}
             - if feature_propose_ao_enabled?
-              p = link_to('Email Parli-branch about this question', "mailto:#{Settings.mail_reply_to}?subject=Question #{question.uin}")
+              - if question.proposed?
+                h3 Deputy Director(s) proposed to answer:
+                p.action_officers-list
+                  - names = question.action_officers.uniq.pluck :name
+                  = names.join(', ')
+
+              p.propose-links
+                = link_to('Email Parli-branch about this question', "mailto:#{Settings.mail_reply_to}?subject=Question #{question.uin}")
```

Comment:
Please replace "this question" with "Question XXX", Or just Email Parli-branch - since I don't think the subject includes anything in the Email anyway.

Reply:
Email subject references the Question in the subject.  

Accessibility issue - I will create a ticket.  Not a blocker.